src/Raven.Server/Web/System/TestConnectionHandler.cs

```diff
@@ -79,12 +77,11 @@ public class TestConnectionHandler : RequestHandler
                     {
                         case TcpConnectionHeaderResponse.AuthorizationStatus.Success:
                             HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
-                            result["Success"] = true;
+                            result[nameof(NodeConnectionTestResult.Success)] = true;
                             break;
                         default:
-                            HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
-                            result["Success"] = false;
-                            result["Error"] = $"Connection to {tcpConnectionInfo.Url} failed because of bad credentials. AuthorizationStatus: {headerResponse.Status}";
+                            result[nameof(NodeConnectionTestResult.Success)] = false;
+                            result[nameof(NodeConnectionTestResult.Error)] = $"Connection to {tcpConnectionInfo.Url} failed because of bad credentials. AuthorizationStatus: {headerResponse.Status}";
```

Comment:
@ayende we know that it's an authorization issue because the AuthorizationStatus of the TcpConnectionHeaderResponse is not "Success", and therefore it's one of the following :          AuthorizationTokenRequired, Forbidden, BadAuthorizationToken, ExpiredAuthorizationToken, ForbiddenReadOnly.
we get no error and there's no other information in the headerRespons.